www/src/components/plugin-searchbar-body.js

```diff
@@ -420,6 +423,20 @@ const Result = ({ hit, pathname, query }) => {
             fontSize: rhythm(4 / 7),
           }}
         >
+          {hit.repository &&
+            hit.repository.url.indexOf(`https://github.com/gatsbyjs/gatsby`) ===
+              0 && (
+              <img
+                src={GatsbyIcon}
+                css={{
+                  width: 20,
+                  marginBottom: 0,
+                  marginRight: 5,
+                  filter: selected ? false : `grayscale(100%) opacity(50%)`,
```

Comment:
```suggestion
                  filter: selected ? false : `grayscale(100%)`,
                  opacity: selected ? false : `50%`,
```
filter is "expensive" I'm unsure if this actually improves anything

Reply:
Further, would it be better to create a separate SVG for the logo in grayscale rather than applying a CSS filter?

Reply:
@wardpeet it should probably be fine here though, right? I presumed that was more a case of images that are very large--is that off base?